nuget/OpenTelemetry.AutoInstrumentation/contentFiles/any/any/integrations.json

```diff
@@ -0,0 +1,362 @@
+[
```

Comment:
The old copy stays in place?
(We can leave with copy if it is easier - will be removed with #2280).

Reply:
For now yes, I am not satisfied with that but I prefer to change that in a separate issue like #2280.